--- /tmp/sol_rpc_docker_build	2026-03-31 13:35:17
+++ scripts/docker-build	2026-03-31 13:42:17
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 # vim: ft=bash
-# Build SOL RPC inside docker. This outputs sol_rpc_canister.wasm.gz.
+# Build ckSOL minter inside docker. This outputs cksol_minter.wasm.gz.
 
 set -euo pipefail
 
@@ -9,7 +9,7 @@
 cd "$SCRIPTS_DIR/.."
 
 function title() {
-    echo "Build SOL RPC canister inside Docker"
+    echo "Build ckSOL minter canister inside Docker"
 }
 
 function usage() {
@@ -24,7 +24,7 @@
 function help() {
     cat << EOF
 
-This will create (and override) "./sol_rpc_canister.wasm.gz".
+This will create (and override) "./cksol_minter.wasm.gz".
 EOF
 }
 
@@ -35,7 +35,7 @@
 
     # image name and build args, made global because they're used in
     # check_feature()
-    image_name="sol-rpc-docker-build"
+    image_name="cksol-docker-build"
     docker_build_args=( --target "scratch_$canister" )
 
     docker_build_args+=(--tag "$image_name" .)
@@ -80,7 +80,7 @@
 done
 
 if [ ${#CANISTERS[@]} -eq 0 ]; then
-    CANISTERS=("sol_rpc_canister")
+    CANISTERS=("cksol_minter")
 fi
 
 for canister in "${CANISTERS[@]}"
