--- /tmp/sol_rpc_build	2026-03-31 13:35:17
+++ scripts/build	2026-03-31 13:47:35
@@ -12,7 +12,7 @@
 #########
 
 function title() {
-    echo "Builds SOL RPC canister"
+    echo "Builds ckSOL minter canister"
 }
 
 function usage() {
@@ -29,7 +29,7 @@
 function help() {
     cat << EOF
 
-Builds the SOL RPC canister.
+Builds the ckSOL minter canister.
 
 NOTE: This requires a working rust toolchain as well as ic-wasm.
 EOF
@@ -57,8 +57,8 @@
             RUN_CARGO_TEST=1
             shift
             ;;
-        --sol_rpc_canister)
-            CANISTERS+=("sol_rpc_canister")
+        --cksol_minter)
+            CANISTERS+=("cksol_minter")
             shift
             ;;
         *)
@@ -120,19 +120,15 @@
             "./target/$TARGET/release/$canister.wasm" \
             -o "./$canister.wasm" \
             shrink
-        ic-wasm "$canister.wasm" -o "$canister.wasm" metadata candid:service -f "canister/$canister.did" -v public
+        ic-wasm "$canister.wasm" -o "$canister.wasm" metadata candid:service -f "minter/$canister.did" -v public
+        ic-wasm "$canister.wasm" -o "$canister.wasm" metadata candid:args -d "(MinterArg)" -v public
 
-        # Indicate the SOL RPC canister init argument type
-        ic-wasm "$canister.wasm" -o "$canister.wasm" metadata candid:args -d "(InstallArgs)" -v public
-
         # Write metadata for dfx
-        asset_name="sol_rpc_canister.wasm.gz"
-        wasm_url="https://github.com/dfinity/sol-rpc-canister/releases/latest/download/sol_rpc_canister.wasm.gz"
-        init_guide="https://internetcomputer.org/docs/current/developer-docs/multi-chain/solana/sol-rpc/sol-rpc-canister"
+        wasm_url="https://github.com/dfinity/cksol/releases/latest/download/$canister.wasm.gz"
+        # TODO: add init_guide once documentation is available
         metadata_json=$(echo '{}' | jq -cMr \
             --arg wasm_url "$wasm_url" \
-            --arg init_guide "$init_guide" \
-            '. | .pullable = { wasm_url: $wasm_url, dependencies: [], init_guide: $init_guide} ')
+            '. | .pullable = { wasm_url: $wasm_url, dependencies: []} ')
         ic-wasm "$canister.wasm" -o "$canister.wasm" metadata dfx -d "$metadata_json" -v public
 
         gzip --no-name --force "$canister.wasm"
